frontend/src/pages/search.tsx

```diff
@@ -17,7 +17,7 @@ export const getServerSideProps: GetServerSideProps = async context => {
   const queryClient = new QueryClient();
   const { initialFiltersStateWithSelectedOptions } = await getInitialFilters(locale, context.query);
   const parsedInitialFiltersState = parseFilters(initialFiltersStateWithSelectedOptions);
-  const initialTextFilter = context.query.text?.toString() ?? null;
+  const initialTextFilter = context.query.text?.toString() ?? '';
```

Comment:
```suggestion
  const initialTextFilter = context.query.text?.toString() || '';
```

_(makes no difference for behavior, but (I think) better for DX)_

Reply:
Anyway, I'm not even sure if this fallback is relevant :D